src/lib/y2network/interface_config_builders/bridge.rb

```diff
@@ -92,8 +101,8 @@ def bridgeable?(iface)
           return false
         end
 
-        if NONBRIDGEABLE_STARTMODE.include?(iface.startmode)
-          log.debug("Excluding (#{iface.name}) - is #{iface.startmode}")
+        if NONBRIDGEABLE_STARTMODE.include?(config.startmode.to_s)
```

Comment:
np: We could add a "bridgeable?" method to the connection.

Reply:
yes and same with bonding, but I am not sure about connection as part of detection is also non existing connection :)